quarkus.ls.ext/com.redhat.quarkus.ls/src/main/resources/com/redhat/quarkus/snippets/quarkus-java.json

```diff
@@ -46,5 +46,23 @@
 		"context": {
 			"type": "io.quarkus.test.junit.SubstrateTest"
 		}
+	},
+	"Quarkus - new integration test class": {
+		"prefix": "qitrc",
+		"body": [
+			"package ${1:packagename};",
+			"",
+			"import io.quarkus.test.junit.QuarkusIntegrationTest;",
+			"",
+			"@QuarkusIntegrationTest",
+			"public class ${TM_FILENAME_BASE} extends ${2:${TM_FILENAME_BASE/^(.*)IT/$1/}Test} {",
+			"",
+			"\t// Execute the same tests, but in native mode.",
+			"}"
+		],
+		"description": "Quarkus native test resource class",
```

Comment:
It should be not `Quarkus new integration test class` ?

Reply:
I've updated it, let me know if it makes sense.

Reply:
Thanks